### DISCUSSION TITLE
Rendering Chinese fails on Chrome when using inline jQuery content?

### DISCUSSION THREAD

**Discussion by shaaraddalvi** [jquery/jquery#4916] [Q&A]
### Description ###
I am trying to see why Chrome and Edge are not able to render Chinese text properly, though the same HTML file is able to render the text properly on Firefox.

Minimal repro content - 
```html
<html>

<body>
    <script>    
      <copy-pasted script content from https://code.jquery.com/jquery-3.6.0.min.js>
	</script>
	<p>
		菩薩慈悲 Some test
	</p>
</body>
</html>
```

Observations till now are - 
* If I copy paste the content of JQuery from https://code.jquery.com/jquery-3.6.0.min.js and paste it inline into the page for use, the Chinese characters break on Chrome.
* If I load JQuery from CDN using the JQuery's suggested snippet 
   ```
   <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
   ```
   it works fine.
* Removing the script tag altogether and just keeping the `<p>` element in the body works fine.

Broken rendered text - 
[![enter image description here][1]][1]

Properly rendered text - 
[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/iLInR.png
  [2]: https://i.stack.imgur.com/tl3pj.png

Also posted on Stackoverflow here - https://stackoverflow.com/questions/68801809/chrome-new-edge-not-rendering-chinese-characters-with-jquery

**Comment by mgol** [comment]
jQuery is not meant to be put in an inline script, we’ve never tested such usage. Let me move that to Discussions.
